projects/vc-api-implementation.md

```diff
@@ -0,0 +1,240 @@
+# Project Name
+_Include the name of project being proposed._
+
```

Comment:
```suggestion
_Include the name of project being proposed._

```
Hi, John.

Thanks for the proposal. You can remove all of these types of items in italics from the proposal. They are intended to be instructional for the proposer.

Reply:
Thanks @tkuhrt . Implemented here: https://github.com/openwallet-foundation/project-proposals/pull/12/commits/fcc0419d666d5dcf585f08903e48a8afc9fdf71e